src/Umbraco.Web/Models/Mapping/CodeFileDisplayMapper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoMapper;
+using Umbraco.Core;
+using Umbraco.Core.Models.Mapping;
+using Umbraco.Core.Models;
+using Umbraco.Web.Models.ContentEditing;
+
+namespace Umbraco.Web.Models.Mapping
+{
+    public class CodeFileDisplayMapper : MapperConfiguration
+    {
+        public override void ConfigureMappings(IConfiguration config, ApplicationContext applicationContext)
```

Comment:
Have verified that the AutoMapperTests pass.

However the test ` Umbraco.Tests.Plugins.PluginManagerTests.Resolves_Trees` Expected 39 but was 37.

Has some trees been removed & merged into one now @hartvig ?

*Forgot how long it takes to run all tests on my machine* - Will update here if I find any other tests that fail